src/chart.zoom.js

```diff
@@ -217,8 +223,17 @@ function panNumericalScale(scale, delta, panOptions) {
 		tickOpts.min = scale.getValueForPixel(scale.getPixelForValue(start) - delta);
 		tickOpts.max = scale.getValueForPixel(scale.getPixelForValue(end) - delta);
 	}
-	tickOpts.min = rangeMinLimiter(panOptions, tickOpts.min);
-	tickOpts.max = rangeMaxLimiter(panOptions, tickOpts.max);
+	
+	var newMax = rangeMaxLimiter(panOptions, tickOpts.max);
+	var newMin = rangeMinLimiter(panOptions, tickOpts.min);
+
+	var limitedNumericDelta = delta < 0 ? newMax - end : newMin - start;
+
+	if (Math.abs(limitedNumericDelta) < 0.1) // Use an epsilon factor to ignore fp rounding errors
```

Comment:
We always include curly braces with our if statements

Reply:
This Pull Request is really old now. I actually thought, that this project is dead and deleted my forked repo. I guess, this PR cannot be updated and I have to create a new PR right?

Reply:
It was sort of dead. I just joined to help revive it and have merged a few PRs today. Yes, as far as I know, you would have to create a new PR. I'm not sure how else to do it unfortunately.

Reply:
Cool. Good to hear, was hoping for some rescuer like you to come :-) 👍 
I'm a bit busy right now, but will look into this in about two weeks.